lib/matplotlib/__init__.py

```diff
@@ -948,7 +948,7 @@ def rc_params_from_file(fname, fail_on_error=False, use_default_template=True):
 
     Parameters
     ----------
-    fname : str
+    fname : str, Path
```

Comment:
On the type description. We have `matplotlib.path.Path`, which is something different, but nevertheless be confusing.

I don't think we have a consistent way to specify `pathlib.Path` object yet. Some places use `path-like` or `PathLike`, which is technically more true, but possibly less understandable for the average user. Maybe we should use `pathlib.Path` here?

Reply:
For the better or the worse (the better, IMO), the stdlib uses "path-like" here (e.g. https://docs.python.org/3/library/functions.html#open); given that for us, this additionally allows disambiguating vs. mpl.path.Path, I think we should standardize to use "path-like" as well.

Reply:
So what needs to be exactly done ? i am not getting how are we goingto standardize path-like

Reply:
We go with
~~~
fname : str or path-like
~~~